frb_codegen/src/generator/dart/ty_enum.rs

```diff
@@ -140,6 +149,9 @@ impl TypeDartGeneratorTrait for TypeEnumRefGenerator<'_> {
                                 .fields
                                 .iter()
                                 .map(|field| {
+                                    if field.name.raw.contains("backtrace") {
```

Comment:
Given that `FrbBacktracedException` indeed only has a field named `backtrace`, maybe we should change "contains" to "=="?